src/CHIPDriver.cc

```diff
@@ -77,7 +77,7 @@ void CHIPReadEnvVarsCallOnce() {
 
   CHIPDeviceTypeStr = readEnvVar("CHIP_DEVICE_TYPE");
   if (CHIPDeviceTypeStr.size() == 0)
-    CHIPDeviceTypeStr = "gpu";
+    CHIPDeviceTypeStr = "default";
```

Comment:
After spending half an hour until realizing pocl was missing SPIR-V, I've spent another half an hour trying to figure out why chipStar still didn't work, until I realized it was filtering out my CPU device.
I've tried to find a good solution but the CL spec was lacking. Would "default" type still work for most people?